tailwind.config.js

```diff
@@ -1,11 +1,15 @@
 /** @type {import('tailwindcss').Config} */
 const { nextui } = require("@nextui-org/react");
 export default {
-  content: ["./src/*.tsx", "./src/**/*.{js,ts,jsx,tsx}", "./node_modules/@nextui-org/theme/dist/**/*.{js,ts,jsx,tsx}"],
+  content: [
+    "./src/*.tsx",
+    "./src/**/*.{js,ts,jsx,tsx}",
+    "./node_modules/@nextui-org/theme/dist/**/*.{js,ts,jsx,tsx}",
+  ],
   theme: {
     colors: {
       turquoise: "#7FB5B5"
```

Comment:
this looks great! thanks for adding colors. can you add a few more so it's easier to build off of?